src/main.rs

```diff
@@ -270,11 +270,12 @@ fn main() {
         }
 
         "new" => {
-            let filetype = matches.value_of("FILETYPE").unwrap();
-            let filename = matches.value_of("FILENAME").unwrap();
+            // TODO: 'cobalt new nonsense' + unwrap = crash so provide default again
+            let filetype = matches.value_of("FILETYPE").unwrap_or("post");
+            let filename = matches.value_of("FILENAME").unwrap_or("new_post.md");
```

Comment:
Why `unwrap_or`? I thought clap provided the default.

I could see at least changing it to `.expect("default value passed to clap")`